pandas/tests/dtypes/test_inference.py

```diff
@@ -235,7 +235,6 @@ class MockFile(object):
     # Iterator but no read / write attributes
     data = [1, 2, 3]
     assert not is_file(data)
-    assert not is_file(mock.Mock())
```

Comment:
What was this changed for?

Reply:
this is the only use of the `mock` fixture within this test so probably unnecessary now.